pushpull/websocket/gateway.py

```diff
@@ -17,7 +17,10 @@
 
 
 async def websocket_rabbitmq_gateway(request):
-    authorization = decode_auth_querystring_param(request.GET)
+    if config.AUTH_COOKIE != '':
+        authorization = request.cookies[config.AUTH_COOKIE];
```

Comment:
When cookie is not provided here, you are now crashing with a `KeyError` , that will result in a 500 response.

It is better to just set authorization to `None` with `request.cookies.get(config.AUTH_COOKIE)` and let the auth module throw a `NotAuthorized` exception later